docs/user/web_data.rst

```diff
@@ -396,37 +396,37 @@ Appendix A: GeoServer Jars
 +------------------------------------------------------------+------------+
 | GeographicLib-Java-1.44.jar                                | 31693      |
 +------------------------------------------------------------+------------+
-| geomesa-accumulo-datastore-1.2.0-SNAPSHOT.jar              | 2338358    |
+| geomesa-accumulo-datastore_2.11-1.3.0-m0.jar               | 3059818    |
```

Comment:
That probably should either 1) come out; 2) be replaced with a hash.  It's also misleading, because if you build from the exact same source, you will get *different* files,  because there are dates inside of the build artifacts.  The file sizes/hashes are only useful for artifacts you download from LocationTech, where they're built exactly once.